proco/schools/tasks.py

```diff
@@ -16,20 +17,20 @@ class FailedImportError(Exception):
 
 
 @app.task(soft_time_limit=30 * 60, time_limit=30 * 60)
-def process_loaded_file(country_pk: int, pk: int):
-    country = Country.objects.get(pk=country_pk)
+def process_loaded_file(pk: int, force: bool = False):
     imported_file = FileImport.objects.filter(pk=pk).first()
     if not imported_file:
         return
 
     imported_file.status = FileImport.STATUSES.started
     imported_file.save()
+    begin_time = timezone.now()
 
     try:
         # todo: rewrite with transaction savepoint
         try:
             with transaction.atomic():
-                warnings, errors = ingest.save_data(country, load_data(imported_file.uploaded_file))
+                warnings, errors = ingest.save_data(imported_file.uploaded_file)
                 if errors:
                     raise FailedImportError
```

Comment:
transaction will be rolled back if file contains errors. force flag should be applied here